great_expectations/dataset/sqlalchemy_dataset.py

```diff
@@ -595,7 +595,7 @@ def expect_column_values_to_be_in_type_list(
                     type_class = getattr(sa, type_)
                     types.append(type_class)
                 except AttributeError:
-                    logger.warning("Unrecognized type: %s" % type_)
```

Comment:
As long as one of the passed types is recognized, this warning does not require any action, so it is better not to emit it.

Reply:
👍